src/20_Components/amp-ad.html

```diff
@@ -53,6 +57,19 @@
     </div>
   </amp-ad>
 
+  <!-- #### Sticky Ads -->
+  <!--
+    `amp-sticky-ad` adds supports for ad units that always take a fixed place in the viewport.
```

Comment:
"adhesion unit" is something very specific (at least to some people) that is different in specifics from `amp-sticky-ad`. Should this be in our examples before we tried it with a few pubs?